lib/teamsnap.rb

```diff
@@ -34,7 +34,7 @@ def init(opts = {})
       self.root_client = TeamSnap::Client.new(:token => token)
 
       ##   Make the apiv3 root call. collection is parsed JSON
-      collection = TeamSnap.run(root_client, :get, "/", {})
+      collection = TeamSnap.run(root_client, :get, self.url, {})
```

Comment:
This should stay as "/".. The line is meant to say hit the root of the url you provided to the init. (the domain is set in the client)